package.json

```diff
@@ -51,5 +54,8 @@
     "io-ts": "1.8.5",
     "italia-ts-commons": "^5.1.11",
     "winston": "^3.2.1"
+  },
+  "resolutions": {
+    "handlebars": "~4.5.3"
```

Comment:
According to the [documentation](https://handlebarsjs.com/api-reference/runtime-options.html#options-to-control-prototype-access), this resolution fixes the generation of code coverage html reports.

Reply:
can you elaborate more on this ?

Reply:
[handlebars](https://handlebarsjs.com/guide/#what-is-handlebars) is hoisted in this project from `istanbul-reports`, it's a templating language used to generate the code coverage reports. With #12 this library was bumped from `4.1.2` to `4.7.3`, but the version `4.6.0` introduced some [breaking changes](https://github.com/wycats/handlebars.js/blob/master/release-notes.md#v460---january-8th-2020), which in this project result in the generation of html pages without data when the `test:coverage` is run, as shown in the image below.
<img width="1017" alt="Schermata 2020-02-19 alle 12 14 43" src="https://user-images.githubusercontent.com/22143453/74829599-a968f980-5311-11ea-8cdc-7626ce1546c5.png">

Some warnings from handlebars are also shown as an output in the console:
```
Handlebars: Access has been denied to resolve the property "statements" because it is not an "own property" of its parent.
You can add a runtime option to disable the check or this warning:
See https://handlebarsjs.com/api-reference/runtime-options.html#options-to-control-prototype-access for details
Handlebars: Access has been denied to resolve the property "branches" because it is not an "own property" of its parent.
You can add a runtime option to disable the check or this warning:
See https://handlebarsjs.com/api-reference/runtime-options.html#options-to-control-prototype-access for details
Handlebars: Access has been denied to resolve the property "functions" because it is not an "own property" of its parent.
You can add a runtime option to disable the check or this warning:
See https://handlebarsjs.com/api-reference/runtime-options.html#options-to-control-prototype-access for details
Handlebars: Access has been denied to resolve the property "lines" because it is not an "own property" of its parent.
You can add a runtime option to disable the check or this warning:
See https://handlebarsjs.com/api-reference/runtime-options.html#options-to-control-prototype-access for details
```
Adding the resolution to version `~4.5.3`  to the `package.json` ensures that the most recent version without the breaking changes will be used, thus allowing the proper generation of the html coverage reports.